src/components/Popover/Arrow.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  Arrow as RadixPopoverArrow,
+  PopoverArrowProps as RadixPopoverArrowProps,
+} from "@radix-ui/react-popover";
+import { vars } from "~/theme";
+import { classNames } from "~/utils";
+import { arrow } from "./Popover.css";
+
+export type PopoverArrowProps = RadixPopoverArrowProps & {
+  backgroundColor?: string;
+  borderColor?: string;
+};
```

Comment:
suggestion: we had a conversation about not using Radix props but rather create own abstraction above radix with our own props